api/Src/Routes/TeamRoutes.ts

```diff
@@ -23,8 +23,11 @@ const teamsRoutes = (controller: TeamController) => {
       .patch(HasRole([UserType.Admin]), controller.updateTeam)
 
     router
-      .route('/teams/:id/users')
+      .route('/teams/:id/user')
```

Comment:
If route is changed here it should be changed immediately on the front in Team.api.ts

Reply:
it will be added by Szymon